packages/core/src/account/base.ts

```diff
@@ -57,7 +75,9 @@ export abstract class BaseSmartContractAccount<
     | PublicErc4337Client<HttpTransport>;
 
   constructor(params: BaseSmartAccountParams<TTransport>) {
-    this.entryPointAddress = params.entryPointAddress;
+    const _entryPointAddress =
+      params.entryPointAddress ?? getDefaultEntryPointContract(params.chain);
+    this.entryPointAddress = _entryPointAddress;
```

Comment:
```suggestion      
    this.entryPointAddress = params.entryPointAddress ?? getDefaultEntryPointContract(params.chain);
```

Reply:
it;s so that I can use _entrypointaddress variabe for getting contract at the end of constructor

Reply:
ah just saw the comment below. sure.